test/recipes/80-test_policy_tree.t

```diff
@@ -18,6 +18,9 @@ use OpenSSL::Glob;
 
 setup("test_policy_tree");
 
+plan skip_all => "No bulk encryption support" if disabled("bulk");
```

Comment:
no-bulk by itself does not mean anything but it implies disabling multiple other things, one of them being ec and that's the reason why the no-bulk builds are broken. So please drop this line.